google_redis/variables.tf

```diff
@@ -0,0 +1,68 @@
+variable "authorized_network" {
+  description = "The network name of the shared VPC"
+  type        = string
+}
+
+variable "custom_name" {
+  default     = ""
+  description = "Use this field to set a custom name for the redis instance"
+  type        = string
+}
+
+variable "application" {
+  description = "Application e.g., bouncer."
+  type        = string
+}
+
+variable "environment" {
+  description = "Environment e.g., stage."
+  type        = string
+}
+
+variable "realm" {
+  description = "Realm e.g., nonprod."
+  type        = string
+}
+
+variable "project_id" {
+  type = string
+}
+
+variable "memory_size_gb" {
+  description = "Memory size in GiB"
+  default     = 1
+  type        = number
+}
+
+variable "redis_configs" {
+  description = "Redis configs https://cloud.google.com/memorystore/docs/redis/reference/rest/v1/projects.locations.instances#Instance.FIELDS.redis_configs"
+  type        = map(string)
+
```

Comment:
I wonder if we should add a default config for activedefrag=on?

Reply:
@jbuck are there any performance considerations or any other reasons why someone would **not** want this enabled? Otherwise I can certainly rework this and add it as a default!

Reply:
I've had it enabled for FxA for years now and not run into any performance problems myself. I tried to look up anything about measuring the performance impact but only found "it's negligible" [here](https://serverfault.com/questions/971804/are-there-situations-were-activedefrag-should-be-kept-disabled-in-redis-5-with), and [here](https://stackoverflow.com/questions/61738210/redis-activedefrag-performance-reduction)

Reply:
Thanks! I added a `merge` and local variable so this is enabled by default. Since it is a `merge` people could override it by passing `{activedefrag: "no"}`

Added in this commit: https://github.com/mozilla/terraform-modules/pull/13/commits/7d3cf6ce102faff9662e7b0363b89825ffe5fb8c